src/js/lib/fileType.test.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+import path from "path"
+
+import lib from "./"
+
+const testFile = (name) => path.join(__dirname, "../../../itest/", name)
+const pcap = testFile("sample.pcap")
```

Comment:
@jameskerr Can I add a pcapng to this for you? I had planned to anyway and this is a good place to start testing it.

Reply:
Yeah, that's fine with me!

Reply:
Added via:
```sh
mergecap -F pcapng -w sample.pcapng sample.pcap
```